src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -1048,5 +1050,15 @@ private void OnContextMenuOpened(object sender, EventArgs e)
             }
             ViewModel.InCanvasSearchViewModel.SearchText = string.Empty;
         }
+
+        private void OnGeometryScaling_Click(object sender, RoutedEventArgs e)
+        {
+            if (GeoScalingPopup == null)
+                GeoScalingPopup = new GeometryScalingPopup(ViewModel.DynamoViewModel);
+
+            GeoScalingPopup.Placement = PlacementMode.Bottom;
+            GeoScalingPopup.PlacementTarget = geometryScalingButton;
```

Comment:
Do we need to set the placement everytime or can we set only once at initialization?

Reply:
I've moved this piece of code inside the if statemente below so it will be executed one time, check the next commit:
https://github.com/DynamoDS/Dynamo/pull/13656/commits/c20979418fa67d8906f184af3810952b71d26641